packages/documentation/src/stories/foundation/layout/Columns/columns.stories.ts

```diff
@@ -0,0 +1,252 @@
+    justifyContent: 'justify-content-start',
+    offsetItem: 'offset-1',
+  },
+  argTypes: {
+    alignItems: {
+      name: 'Align Items',
+      description: 'Aligns the whole row vertically.',
+      control: {
+        type: 'select',
+      },
+      options: ['align-items-start', 'align-items-center', 'align-items-end'],
+      table: {
+        category: 'General',
+      },
+    },
+    alignSelf: {
+      name: 'Align Item 1',
+      description: 'Aligns the Item 1 vertically.',
+      control: {
+        type: 'select',
+      },
+      options: ['no self alignment', 'align-self-start', 'align-self-center', 'align-self-end'],
+      table: {
+        category: 'General',
+      },
+    },
+    justifyContent: {
+      name: 'Horizontal Alignement',
+      description: 'Aligns the Items horizontally.',
+      control: {
+        type: 'select',
+      },
+      options: [
+        'justify-content-start',
+        'justify-content-center',
+        'justify-content-end',
+        'justify-content-around',
+        'justify-content-between',
+        'justify-content-evenly',
+      ],
+      table: {
+        category: 'General',
+      },
+    },
+    offsetItem: {
+      name: 'Offset classes',
+      description: 'Increases the left margin of a column',
+      control: {
+        type: 'select',
+      },
+      options: [
+        'offset-1',
+        'offset-2',
+        'offset-3',
+        'offset-4',
+        'offset-5',
+        'offset-6',
+        'offset-7',
+        'offset-8',
+        'offset-9',
+        'offset-10',
+        'offset-11',
+      ],
+      table: {
+        category: 'General',
+      },
+    },
+  },
+};
+
+export default meta;
+
+type Story = StoryObj;
+
+export const VerticalExample: Story = {
+  parameters: {
+    controls: { exclude: ['Horizontal Alignement', 'Offset classes'] },
+  },
+  render: (args: Args) => html`
+    <div class="container">
+      <div class="row-height row ${args.alignItems}">
+        <div class="col${args.alignSelf === 'no self alignment' ? '' : ` ${args.alignSelf}`}">
+          Item 1
+        </div>
+        <div class="col">Item 2</div>
+        <div class="col">Item 3</div>
+      </div>
+    </div>
+  `,
+};
+
+export const HorizontalExample: Story = {
+  parameters: {
+    controls: { exclude: ['Align Items', 'Align Item 1', 'Offset classes'] },
+  },
+  render: (args: Args) => html`
+    <div class="container">
+      <div class="row ${args.justifyContent}">
+        <div class="col-4">Item 1</div>
+        <div class="col-4">Item 2</div>
+      </div>
+    </div>
+  `,
+};
+
+export const OrderExample: Story = {
+  render: () => html`
+    <div class="container">
+      <div class="row">
+        <div class="col">First in DOM, no order applied</div>
+        <div class="col order-5">Second in DOM, with a larger order</div>
+        <div class="col order-1">Third in DOM, with an order of 1</div>
+      </div>
+    </div>
+  `,
+};
```

Comment:
Can we add some controls to make this interactive? This would be much easier to understand, if user can switch some of the `order` classes.

We could for example add a dropdown select control to set the order-class on each column.
e.g. Column1Order, Column2Order, Column3Order...